src/test/java/it/TestApplication.java

```diff
@@ -17,7 +17,7 @@
 
 import org.junit.Test;
 
-public class TestApplication extends EndpointTest {/*
+public class TestApplication extends EndpointIT {/*
```

Comment:
remember that surefire is automatically bound to the war maven lifecycle even though it is not in your pom.  Looking at the lifecycle in eclipse helps.